### ISSUE TITLE
List of primitive type in schema

### ISSUE THREAD

**Issue by fever324** [realm/realm-js#860]
Hi,

How can I declare a schema that have a field that is a list of doubles?
I tried the following schema, but it's not working.
```
const TestSchema = {
  name: 'Test',
  primaryKey: 'id',
  properties: {
    id: 'string',
    something: { type:'list', objectType:'double' },
  },
}
```

![image](https://cloud.githubusercontent.com/assets/3506383/22870653/32dcfc84-f177-11e6-81e8-751e3f9173a5.png)

**Comment by kristiandupont** [comment]
Currently, lists of primitive types are not supported. You will have to declare a class that contains a `double` property and create the list out of that. Suboptimal yes, and we are working on improving this.

**Comment by EliCDavis** [comment]
I apologize for bumping a closed issue, but should this not be kept open and labeled as something like a feature?

**Comment by kristiandupont** [comment]
You are right, and work is being done on this now.

**Comment by kneth** [comment]
It has been merged into `2.0.x`
